src/api/handlers/projects/jobs.py

```diff
@@ -695,6 +695,9 @@ def get(self, project_id, job_id):
 class ArchiveDownload(Resource):
 
     def get(self, project_id, job_id):
+        token = request.headers.get('Authorization')
+        if token:
+            token = token.replace("bearer ", "")
```

Comment:
what if it starts with Bearer

Reply:
Hi @iberryful  Thanks for your suggersions. Now it could recognize Bearer.